conda/recipes/dask-cuml/meta.yaml

```diff
@@ -34,7 +34,7 @@ requirements:
     - cudf
     - dask >=0.19.0
     - distributed >=1.23.0
-		- cuml
+		- dask-cuml
```

Comment:
I don't think this right. You shouldn't depend on yourself to run.

Also, looking at the full file, the build dependencies are really only on python? You probably need dask, cudf, etc to build right?